build.gradle

```diff
@@ -93,7 +214,22 @@ subprojects {
     }
     dependencies {
         /* Standard libraries */
+        ballerinaStdLibs "org.ballerinalang:http-ballerina:${stdlibHttpVersion}"
+        ballerinaStdLibs "org.ballerinalang:mime-ballerina:${stdlibMimeVersion}"
+        ballerinaStdLibs "org.ballerinalang:auth-ballerina:${stdlibAuthVersion}"
+        ballerinaStdLibs "org.ballerinalang:file-ballerina:${stdlibFileVersion}"
+        ballerinaStdLibs "org.ballerinalang:cache-ballerina:${stdlibCacheVersion}"
+        ballerinaStdLibs "org.ballerinalang:crypto-ballerina:${stdlibCryptoVersion}" //
         ballerinaStdLibs "org.ballerinalang:encoding-ballerina:${stdlibEncodingVersion}"
+        ballerinaStdLibs "org.ballerinalang:task-ballerina:1.1.0-SNAPSHOT"
```

Comment:
Shall we use the constant here for the version?

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-websub/pull/17/commits/dffc9024569233a20a1e9b4b81cc35ed80ce5024